### DISCUSSION TITLE
nrf52840 and ENC28J60 Interfacing Issues

### DISCUSSION THREAD

**Discussion by RaysoftTechnologiesInc** [zephyrproject-rtos/zephyr#61223] [General]
Hi guys, 

I realized the issue I opened earlier #61190 had been closed before I got a resolution to the problem. I am still having the same error in building even after having tried to follow the instructions in the responses i got.

Could you please give me some pointers as to where I may be getting lost?

![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/8f41f456-a851-4a91-a8c1-4132e922f8b6)
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/18cba4ea-193d-474d-88db-7d5537e7aa7f)
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/57799192-5c81-41ce-9d15-f81b1e909457)
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/e9df95c7-f5e9-457c-acac-cab665829c82)
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/35f7afdf-985f-4983-94bf-91b2aff286dc)

**Comment by nordicjm** [comment]
The issue was closed because it is not a bug, this is user error. I will move it to a discussion

**Comment by nordicjm** [comment]
You need to enable the driver to be used, if you run `ninja menuconfig` you will most likely need to to to drivers, enable SPI, go back, go to networking, go to devices, enable ethernet, go back to the root, go to drivers, go to ethernet, and ensure the 28j60 driver is selected, then if you save and exit, it should build.

**Reply by RaysoftTechnologiesInc** [reply]
running the ninja menuconfig gives me this error
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/6a38eead-8ebe-434d-96f9-ee09fa0d621a)

**Reply by nordicjm** [reply]
you need to be in the build directory to run that

**Reply by RaysoftTechnologiesInc** [reply]
Hi @nordicjm,

Thank you, i got somewhere, i can get it to compile now. However i am getting this error of not getting any network interface. 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/45296244/95c95171-0b15-4c74-92b4-a35e83029cc0)

Do you know where i may have missed something?

**Reply by nordicjm** [reply]
Is enc28j60 enabled under drivers > ethernet?
